<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>C3 Molecular origins of diverse cancers • YESCDS</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="C3 Molecular origins of diverse cancers">
<meta property="og:description" content="YESCDS">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">YESCDS</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.0.30</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/A0_intro_notebook.html">A0 Introductory Notebook</a>
    </li>
    <li>
      <a href="../articles/A1_cancer_table.html">A1 Cancers in my world</a>
    </li>
    <li>
      <a href="../articles/A2_rates.html">A2 Cancer rates for public health assessment</a>
    </li>
    <li>
      <a href="../articles/A3_monitoring.html">A3 Monitoring of cancer by public health institutions</a>
    </li>
    <li>
      <a href="../articles/A4_interactive_tables.html">A4 Using interactive tables</a>
    </li>
    <li>
      <a href="../articles/A5_data_graphics.html">A5 Data graphics</a>
    </li>
    <li>
      <a href="../articles/A6_standardization.html">A6 Age standardization of cancer rates</a>
    </li>
    <li>
      <a href="../articles/B1_global.html">B1 Cancer across the globe</a>
    </li>
    <li>
      <a href="../articles/B2_table_plus_geo.html">B2 Enhancing tables with geography</a>
    </li>
    <li>
      <a href="../articles/B3_counties.html">B3 Cancer in our counties</a>
    </li>
    <li>
      <a href="../articles/C1_body_overall.html">C1 Cancer in our bodies: Overall view</a>
    </li>
    <li>
      <a href="../articles/C2_variation_survival.html">C2 Variation in survival with cancer</a>
    </li>
    <li>
      <a href="../articles/C3_subtypes_molecular.html">C3 Molecular origins of diverse cancers</a>
    </li>
    <li>
      <a href="../articles/D1_clin_trial_concepts.html">D1 Clinical trials concepts</a>
    </li>
    <li>
      <a href="../articles/D2_equipoise.html">D2 Equipoise: when medical knowledge can only advance by experimenting</a>
    </li>
    <li>
      <a href="../articles/D3_randomization_bias.html">D3 Randomization to reduce bias</a>
    </li>
    <li>
      <a href="../articles/D4_comparing_trt.html">D4 Methods of comparing treatments</a>
    </li>
    <li>
      <a href="../articles/E1_heidelberg2022.html">E1 Inclusive genomic data science education, illustrated with cancer genomics</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/vjcitn/YESCDS/">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>C3 Molecular origins of diverse cancers</h1>
                        <h4 data-toc-skip class="author">Vincent J. Carey, stvjc at channing.harvard.edu</h4>
            
            <h4 data-toc-skip class="date">September 15, 2022</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/vjcitn/YESCDS/blob/HEAD/vignettes/C3_subtypes_molecular.Rmd"><code>vignettes/C3_subtypes_molecular.Rmd</code></a></small>
      <div class="hidden name"><code>C3_subtypes_molecular.Rmd</code></div>

    </div>

    
    
<pre><code><span><span class="co">## Warning: multiple methods tables found for 'aperm'</span></span></code></pre>
<pre><code><span><span class="co">## Warning: replacing previous import 'BiocGenerics::aperm' by</span></span>
<span><span class="co">## 'DelayedArray::aperm' when loading 'SummarizedExperiment'</span></span></code></pre>
<div class="section level2">
<h2 id="molecular-origins-of-tumors-and-their-spread">Molecular origins of tumors and their spread<a class="anchor" aria-label="anchor" href="#molecular-origins-of-tumors-and-their-spread"></a>
</h2>
<p>In this segment we will review basic definitions underlying the use of data to understand variation in genetic processes connnected with initiation and progression of cancer.</p>
<div class="section level3">
<h3 id="human-genome">Human genome<a class="anchor" aria-label="anchor" href="#human-genome"></a>
</h3>
<p>The human genome is composed of 22 autosomal chromosome pairs and 1 pair of sex chromosomes. Autosomal chromosomes are numbered 1-22; females have a pair of X chromosomes, and males have a pair labeled X and Y.</p>
<div class="figure">
<img src="images/chromosomes.jpg" alt="Genome overview"><p class="caption">Genome overview</p>
</div>
</div>
<div class="section level3">
<h3 id="central-dogma">Central dogma<a class="anchor" aria-label="anchor" href="#central-dogma"></a>
</h3>
<p>“DNA makes RNA makes protein”</p>
<p>The genetic material from which chromosomes are formed is DNA (deoxyribonucleic acid).</p>
<p>DNA is transcribed to RNA (ribonucleic acid)</p>
<p>RNA is translated to protein.</p>
<p>A schematic from <a href="https://en.wikipedia.org/wiki/Central_dogma_of_molecular_biology#/media/File:Cdmb.svg" class="external-link">Wikipedia</a>, by Adenosine</p>
<div class="figure">
<img src="images/centralDogma.jpg" alt="Central dogma"><p class="caption">Central dogma</p>
</div>
</div>
<div class="section level3">
<h3 id="gene-counts-by-chromosome">Gene counts by chromosome<a class="anchor" aria-label="anchor" href="#gene-counts-by-chromosome"></a>
</h3>
<p>“Gene” refers to a physical and functional component of the genome that is inherited between generations.</p>
<p>Gene finding and gene counting are basic tasks of bioinformatic analysis.</p>
<p>The sizes of chromosomes and numbers of genes currently known are depicted here:</p>
<div class="figure">
<img src="images/genecounts.jpg" alt="Gene counts"><p class="caption">Gene counts</p>
</div>
</div>
<div class="section level3">
<h3 id="browsing-the-reference-genome">Browsing the reference genome<a class="anchor" aria-label="anchor" href="#browsing-the-reference-genome"></a>
</h3>
<p>Visit <a href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&amp;lastVirtModeType=default&amp;lastVirtModeExtraState=&amp;virtModeType=default&amp;virtMode=0&amp;nonVirtPosition=&amp;position=chr2%3A179250000%2D179812868&amp;hgsid=1377911977_sxQEMUOHBeopzad7Oql0bsSeaceA" class="external-link">the genome browser for the gene called TTN</a> and you will see something like</p>
<div class="figure">
<img src="images/genomeBrowser.jpg" alt="browser"><p class="caption">browser</p>
</div>
<p>The display includes a karyogram (depiction of the entire chromosome) with a red patch indicating the portion of the chromosome being viewed. You can zoom out or in using the controls.</p>
<p>Each of the bands labeled TTN depicts a transcript of that gene. Distinct transcripts are formed as transcriptional machinery splices out different subsequences that we call introns. The transcript is a series of stitched-together exons.</p>
<div class="section level4">
<h4 id="exercises">Exercises<a class="anchor" aria-label="anchor" href="#exercises"></a>
</h4>
<p>C.3.1 Use the UCSC genome browser to visualize the entirety of chromosome 17 in reference build “hg19”. What is the length of the chromosome in base pairs?</p>
<p>C.3.2 Browse to the gene BRCA2. What is its length in base pairs?</p>
</div>
<div class="section level4">
<h4 id="answers">Answers<a class="anchor" aria-label="anchor" href="#answers"></a>
</h4>
<pre><code><span><span class="va">C.3.1</span></span>
<span></span>
<span><span class="va">C.3.2</span></span></code></pre>
</div>
</div>
<div class="section level3">
<h3 id="copy-number-alterations">Copy number alterations<a class="anchor" aria-label="anchor" href="#copy-number-alterations"></a>
</h3>
<p>Changes to chromosome structure are often found in tumor DNA when compared to normal tissue DNA. An example is region duplication:</p>
<div class="figure">
<img src="images/CNAdup.jpg" alt="CNA"><p class="caption">CNA</p>
</div>
<p>Deletions and translocations (parts of chromosomes are exchanged) are other important genomic alterations in cancer.</p>
</div>
</div>
<div class="section level2">
<h2 id="views-of-genomic-variation-in-breast-cancer">Views of genomic variation in breast cancer<a class="anchor" aria-label="anchor" href="#views-of-genomic-variation-in-breast-cancer"></a>
</h2>
<div class="section level3">
<h3 id="a-2012-view-of-the-whole-genome">A <a href="http://www.ncbi.nlm.nih.gov/pubmed/22522925" class="external-link">2012 view</a> of the whole genome<a class="anchor" aria-label="anchor" href="#a-2012-view-of-the-whole-genome"></a>
</h3>
<p>In this display, the chromosomes are ordered across the screen, from 1 to 22 followed by X.</p>
<p>Because this display concerns breast cancer in females, no Y chromosome is depicted.</p>
<p>The display is focused on “copy number alterations” (CNAs) in tumors – where (possibly large) segments of chromosomal DNA have been duplicated or deleted in the tumor. Red spikes denote positions where duplicating CNAs are associated with expression changes of nearby genes, blue spikes where a deleting CNA affects expression.</p>
<div class="figure">
<img src="images/shahGenomeWide.jpg" alt="Genome-wide drivers"><p class="caption">Genome-wide drivers</p>
</div>
<p>The figure above is drawn from a paper studying 2000 breast tumors. A more recent <a href="https://www.frontiersin.org/articles/10.3389/fonc.2021.700568/full" class="external-link">survey</a> shows that the phenomenon pervades a broad spectrum of cancer types studied in The Cancer Genome Atlas (TCGA).</p>
<div class="figure">
<img src="images/COSMICCNA.jpg" alt="2021 CNA survey"><p class="caption">2021 CNA survey</p>
</div>
</div>
<div class="section level3">
<h3 id="tcga-common-mutations-ttn-mutation-and-survival">TCGA: common mutations; TTN mutation and survival<a class="anchor" aria-label="anchor" href="#tcga-common-mutations-ttn-mutation-and-survival"></a>
</h3>
<p>This display is from the NCI Genomic Data Commons:</p>
<div class="figure">
<img src="images/breastTTN.jpg" alt="Breast cancer summary"><p class="caption">Breast cancer summary</p>
</div>
<p>While the “titin” gene (TTN) is frequently mutated in TCGA breast cancer samples, and there is some indication that TTN mutations are disadvantageous for survival, the mechanism by which TTN may affect tumor progression is not understood.</p>
<p>The mutations seen in TTN may be “passenger” mutations with no functional significance.</p>
<p>The mutation “spectrum” is sometimes depicted using the oncogrid display.</p>
<div class="figure">
<img src="images/oncogridBRCA.jpg" alt="OncoGrid"><p class="caption">OncoGrid</p>
</div>
<p>Each column is a single tumor; each row a gene in which mutations were found for at least one of the patients. The overall interpretation is that there is striking diversity in the collection of mutated genes found for each patient.</p>
<div class="section level4">
<h4 id="exercises-1">Exercises<a class="anchor" aria-label="anchor" href="#exercises-1"></a>
</h4>
<p>C.3.3 Use the Genomic Data Commons data portal to explore data on prostate cancer in the TCGA project. What are the five genes reported to present the largest number of mutations across patients in that project?</p>
<ul>
<li>Repeat the exercise for ovarian cancer in TCGA.</li>
</ul>
<p>C.3.4 Use the “Survival” widget at the Genomic Data Commons to compare the survival times for ovarian cancer patients with and without mutation in CSMD3. Is CSMD3 mutation associated with longer or shorter median survival time? About how large is the difference?</p>
</div>
<div class="section level4">
<h4 id="answers-1">Answers<a class="anchor" aria-label="anchor" href="#answers-1"></a>
</h4>
<pre><code><span><span class="va">C.3.3</span></span>
<span></span>
<span><span class="va">C.3.4</span></span></code></pre>
</div>
</div>
<div class="section level3">
<h3 id="clustering-mrna-profiles-of-genes-and-tumors-to-find-subtypes-van-t-veer-et-al--2002">Clustering mRNA profiles of genes and tumors to find subtypes: <a href="https://www.nature.com/articles/415530a.pdf?origin=ppub" class="external-link">van ’t Veer et al. 2002</a><a class="anchor" aria-label="anchor" href="#clustering-mrna-profiles-of-genes-and-tumors-to-find-subtypes-van-t-veer-et-al--2002"></a>
</h3>
<p>The following display summarizes about 500000 numbers (5000 genes by 98 tumors). Red patches are plotted for genes that are up-regulated (compared to adjacent normal tissue) in the associated tumor. Cluster trees are included to show the existence of groups genes that have similar expression across tumors, and groups of tumors that show similar expression across genes.</p>
<div class="figure">
<img src="images/vantveerHeat.jpg" alt="Classic heatmap"><p class="caption">Classic heatmap</p>
</div>
</div>
</div>
<div class="section level2">
<h2 id="evolution-of-tumor-genome-within-a-patient-with-metastatic-breast-cancer">Evolution of tumor genome within a patient with metastatic breast cancer<a class="anchor" aria-label="anchor" href="#evolution-of-tumor-genome-within-a-patient-with-metastatic-breast-cancer"></a>
</h2>
<p>Most of what we have covered so far pertains to molecular aspects of tissue taken from primary tumor samples.</p>
<p>After time, and especially after treatment has begun, tumor genomes can adapt to avoid treatment effects and resume division.</p>
<p>A <a href="https://genomemedicine.biomedcentral.com/track/pdf/10.1186/s13073-021-00989-6.pdf" class="external-link">2021 paper from the Gabor Marth lab</a> details the experience of one patient.</p>
<div class="figure">
<img src="images/marthPatient.jpg" alt="Patient history"><p class="caption">Patient history</p>
</div>
<p>Using a rapid autopsy and genetic sequencing of tumors that metastasized to multiple organs, Huang and colleagues map the clonal diversification of the primary tumor genome into tumors that emerged in different organs.</p>
<div class="figure">
<img src="images/marthEvolve.jpg" alt="Evolutionary map"><p class="caption">Evolutionary map</p>
</div>
<p>One of the elements with which this map of progressive metastasis was constructed is an analysis of copy number aberrations (CNAs). An interactive view of these CNAs in sampes from the 28 tumors in different body site is <a href="cnv-heatmap.html">here</a>.</p>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Vince Carey.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.6.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
